docs/en/02_Developer_Guides/00_Model/10_Versioning.md

```diff
@@ -129,6 +129,77 @@ is initialized. But it can also be set and reset temporarily to force a specific
 	$obj = MyRecord::getComplexObjectRetrieval(); // returns 'Stage' records
 	Versioned::set_reading_mode($origMode); // reset current mode
 
+### File ownership
+
+Typically when publishing versioned dataobjects, it is necessary to ensure that some linked components
+are published along with it. Unless this is done site front-end content can appear incorrectly published.
+
+For instance, a page which has a list of rotating banners will require that those banners are published
+whenever that page is.
+
+The solution to this problem is the ownership API, which declares a two-way relationship between
+objects along database relations. This relationship is similar to many_many/belongs_many_many
+and has_one/has_many, however it relies on a pre-existing relationship to function.
+
+For instance, in order to specify this dependency, you must apply `owns` and `owned_by` config
+on a relationship.
+
+
+	:::php
+	class MyPage extends Page {
+		private static $has_many = array(
+			'Banners' => 'Banner'
+		);
+		private static $owns = array(
+			'Banners'
+		);
+		
+		public function onAfterVersionedPublish($fromStage, $toStage, $createNewVersion) {
```

Comment:
Assuming the CMS will take care of this automatically through changesets, is it helpful to add a custom "auto-publish" implementation in this high level documentation? It gives the impression that you need to write custom hooks for this feature to work.